test/Universe.js

```diff
@@ -562,6 +562,13 @@ contract(
       expect(cid[0]).to.eq(contractId("Universe"));
     });
 
-    it("should revert on setting KNOWN_INTERFACE_UNIVERSE singleton");
+    it("should revert on setting KNOWN_INTERFACE_UNIVERSE singleton", async () => {
+      await expect(universe.setSingleton(knownInterfaces.universe, universe.address)).to.revert;
```

Comment:
this revert because it's not called from
```
 {
           from: universeManager,
         },
```
please add it to test. that's why revert codes are so important addition. you can add such revert code to the contract